### ISSUE TITLE
Window app moves to background process when the app is minimized

### ISSUE THREAD

**Issue by queuestack** [social-manager-tools/social-manager-tools#46]
### Versions

Replace the values below with your own:

- **Social Manager Tools Version:** v0.6.1
- **Social Platform:** Instagram
- **Operating System:** Windows 10

### Expected Behavior

When I see the process in window task manager, the app is still running when the app is minimized, and reopens.

### Actual Behavior

The app moves to background process when it is minimized, so I can't see the UI of the app.

```

```

### Steps to Reproduce

Please include the steps the reproduce the issue, numbered below. Include as
much detail as possible.

1. Open the app in Windows 10
2. Open the process tab in task manager 
3. Minimize the app
4. You can see the app is now running on the background process

### config.js (remember to remove password)
```

```

### Screenshots (Optional)
![Screenshot 1](https://lh3.googleusercontent.com/DR5M1_eqDUOlzOzAcOyPgZaZ7XaOIDQDNb4dBVqibEpvJfZoMQoAfQWJYl89c284tq3vGK5UZDm9aGpQUoCXwft-qpTM91u87Bq0FYeWi7UNoA2v8j7eIm5lZMyuXwf9Y4y8zDJfLKODN9NqfRRa7q7j--Y50P-ols7P63KctzmzvbjutWSwnTqTyRzQrxMsF3579LJZGtLTSkEtj9G62Ii4ko_CGy-pLspDBoUJwfqW_WditL1o1aUSqw_qPa1H5QlBY90Q7FN-ndcI3FHY7l9m4wQRQvwFEJGAayCk50NuQwpcbPwN4N6TccWxBFwx2nSNvYJe2nNIqGEWy_BeGMHyP_DW_yV8k2gRs8AdG9vfJtBIbN_HfjRAHLSkatnlqFL5PEGUw9gXzTWcZNiFrM2Xt_W5-vi3f83b0tMgG25AIz1Tq1hmqCbv9TQC2_mE3aSYah-NaZD6jYNPlFQWU4x6qC4UIiP5QkOWUQaAYkgL-_3iWPqSq2INqQDo1FwmDwycYj0IL68NeRkX7va49bxKgvabj-Tpl0npt4i8TXyelME4G-4ILQ5M5zb-aF2ZGlHa_aR8ushO6CebLAUh9MaApuJrJy62QdwGlbEE7OIhhxLJ7Zo4N2o6sIzdgPII3mn5GntZI7nNBL0dHcJFiDPylqvXa_7WpUBUqsx-SJSJomoI4Nt6TGUB44JqSwc7TbKGFUL6HtROPBZyFLkWDl0pi9Mkuw1XSawqaP8=w910-h512-no)

![Screenshot 2](https://lh3.googleusercontent.com/PQofa7wzJzKthMxu0A8X8cZkzkae35tLGWa2DmCOZ4gccluDtc4XIBHf4_pmFRwQwm4e_gPCW_68IKfYRwKQaUv_09z0V6Dsaiq-uEQ75R_LtgPPtufW1KQ7oVa6wWkUznTG0GXqJbY4dK0sLk2XJ4uGY8Tmsm04AHlwN5acaIe9rP8MeL0FDSr3C715LZQu8eZC8-lzbpiGllODkU7HRNW6zFnez0UCD6JEquPHX8GWh4mW-kMgI8nfVwAurb8ab1U1WV-hqJ4nAyBxqsy25ybN5xJobeqse-fQgW_-5CVmZcp83fFF0dFf0OZG4JbAF2hmpJp45-PBfMejxXX650zS_PmiAYl8ipaYhuLboThgCHuZe2TX43u4WYt-sfmvr7kSj4e7uQtymMppOkIoXTrsvyIlYtrJodQynk7l4UdORRhdRuDBcInBYHSNc8CUFL3zkLQE8SliSegtmjzIU2-zovj1R8bZj9CEfly-M7iV-kVxNVOqAH3BlEYz7TojdbL_m_YACddf7Q9leDGbZskyqg_K4MFfv7NpM7bSQEYytKeYOw4DbM0G0R01Ho_GrmU3HeuWOgd5c__hKTdDPpEme5-2b1Idz3DIWPy8cctf2mK0t5FyE8W_f4tjbSh9kjM9CVb9LWii-hV8QZgTr7FT8jEWW42oi_GjSgLZS4m-av8XPDL0er_BZygTcdP_LzVNBDUuPw8KinBJml2gYwk6kd4KHGXIzu3bU3U=w910-h512-no)

If the error is graphical in nature it is helpful to provide a screenshot.

**Comment by queuestack** [comment]
I found the problem is caused by `main_window.hide()` on 'minimize' event handler at main.js. When I comment the hide line, the app works as I expected.

main.js
``` js
...
    if (process.platform != "darwin") {
        app_icon.on("click", function(event) {
            event.preventDefault();
            if (main_window.isVisible()) {
                main_window.hide();
            } else {
                main_window.show();
            }
        });

        main_window.on("minimize", function(event) {
            event.preventDefault();
            main_window.hide();
        });

        // main_window.on("show", function() {
        //     app_icon.setHighlightMode("always");
        // });

        app_icon.setContextMenu(context_menu);
    }
...
```

**Comment by ptkdev** [comment]
You are sure try icon work again? 
If you minimize/maximize work again?
If you click on try icon for open/minimize app work again?

If work send pull request :)

**Comment by ptkdev** [comment]
Fix available on `v0.6.2`
